.env.sample

```diff
@@ -1,4 +1,5 @@
 DB_NAME=cuuhomientrung
-DB_USER=postgres
```

Comment:
@dshongphuc I don't see any reason we have to include the change of configuration example in this pull request. Can you exclude it?